cypress/tests/core/blocks/blocks-autofocus.js

```diff
@@ -0,0 +1,172 @@
+describe('New Block Auto Focus Tests', () => {
+  beforeEach(() => {
+    // given a logged in editor and a page in edit mode
+    cy.visit('/');
+    cy.autologin();
+    cy.createContent({
+      contentType: 'Document',
+      contentId: 'my-page',
+      contentTitle: 'My Page',
+    });
+    cy.visit('/my-page');
+    cy.waitForResourceToLoad('@navigation');
+    cy.waitForResourceToLoad('@breadcrumbs');
+    cy.waitForResourceToLoad('@actions');
+    cy.waitForResourceToLoad('@types');
+    cy.waitForResourceToLoad('my-page');
+    cy.navigate('/my-page/edit');
+  });
+
+  it('Press Enter on a description block adds new autofocused default block', () => {
+    cy.intercept('GET', '/**/my-page').as('content');
+    cy.intercept('PATCH', '*').as('save');
+    // when I add a text block
+    //add listing block
+    cy.getSlate().click();
+    cy.get('button.block-add-button').click();
```

Comment:
I don't understand cypress, but it seems that this lines can stay in the `beforeEach` function, since they are the same in all tests.